src/extension/core/src/main/java/au/org/emii/core/Config.java

```diff
@@ -0,0 +1,153 @@
+package au.org.emii.core;
```

Comment:
I was planning to just update the wps extension and not touch ncwms for this task as it is out of  scope. Also, probably I will need to setup test data/environment for ncwms before updating it.